src/FuncHW/Program.cs

```diff
@@ -25,18 +25,25 @@ static void Main()
 
             List<Cat> cats = new() { barsik, pushok, murzik, dymok, murka };
 
-            //Func*** funcPersonIsActive = ***;
```

Comment:
Rollback this comments, please

You can delete only commented-functions that you implemented

Reply:
This is necessary, because, when your merge request will be merged, all other students will not see theirs tasks

Reply:
Sorry, returned comments